### PR TITLE
Write code with using Tidyverse

### PR DESCRIPTION
**Pull request by TrangQuach**
Dear William,

I write the code using Tidyverse to do under your code. The output of code is html file. I tried to create a word document. However, the pictures are too big, so they do not fit in word document. 

Only two files are changed, which are your R code (RMD) and output of R code (HTML).

Thanks,
Trang

### CODE REVIEW

**Comment by williamnorfolk** [comment]
Trang there is a merge conflict with our two repositories, I am not quite sure as to why but I am investigating a bit currently. If you have any idea please let me know!

**Comment by TrangQuach** [comment]
Hi William,

I do not have the real solution for this situation. My solution is that you close my request for merging. Then, I try to do it again. It means:

I will fork your repository again. Then, write the code and run code to produce the output. Finally, do "pull request" again. What do you think?

**Comment by williamnorfolk** [comment]
Hi Trang,

I think I was able to figure out the issue with the merge, I had an orphaned change on my local repo that had not been pushed to master (I am not really sure what was different about it but GitKraken was recognizing it as an unpushed change).  I was able to push that change through to the master now, but I think you need to re-fork the repository (make sure you save all of you Tidyverse exercise work somewhere before hand) and then re-paste your new code back into the Rmd from the newly forked repository. Does that make sense? 

Basically the fork you have is now one commit behind the master (sorry I did not even know the change was unstaged) so if you create a new fork, paste all of the changes you previously made back into the new Rmd I think it should work!

**Comment by williamnorfolk** [comment]
Yes perfect Trang! Sorry I must have been typing my comment while you were doing the same! I think a re-fork and merge should do it!

**Comment by TrangQuach** [comment]
That's ok for me. Let me know when you finish your uploading. Please close my merging request. Then, I go ahead with re-doing.

**Comment by williamnorfolk** [comment]
Alright should be all good to go now (hopefully) for a re-fork, I will close this pull-request so you can submit a new one!
